src/Repository/Repository.csproj

```diff
@@ -4,7 +4,7 @@
 
   <PropertyGroup>
     <Description>Repository to hold objects which can be identified by an integer.</Description>
-    <TargetFramework>netstandard1.0</TargetFramework>
```

Comment:
I think `TargetFramework` should remain `netstandard1.0` since no dependencies require higher than this.